src/base/Context.ts

```diff
@@ -523,6 +523,30 @@ export class Context {
     return invalidDids;
   }
 
+  /**
+   * Returns an Identity when given a did string
+   *
+   * @throws if the Identity does not exist
+   */
+  public async getIdentity(identity: Identity | string): Promise<Identity> {
+    let id;
+    if (typeof identity === 'string') {
+      id = new Identity({ did: identity }, this);
+    } else {
+      id = identity;
+    }
```

Comment:
Here we should just return. If the user has an Identity instance, it's because it was already checked for existence somewhere else, or directly returned from the chain. Since Identities cannot be removed, the check would be redundant